config.json

```diff
@@ -1,5 +1,6 @@
 {
-    "bootstrap_path": "/bower_components/bootstrap",
+    "bower_path": "../../bower_components",
```

Comment:
do we still  need to have the two separate config options here for bower and bootstrap?